packages/formation-react/src/components/MegaMenu/MegaMenu.mdx

```diff
@@ -154,7 +154,7 @@ This will generate a dropdown with only links.
 ```
 **This will generate this view.**
 
-![alt nav style 1](../../../../documentation/src/images/mega-menu/nav-version-2.png)
+![alt nav style 1](../../../documentation/src/images/mega-menu/nav-version-2.png)
```

Comment:
fixed reference to this image- we may want to tweak webpack resolvers to work with just `documentation/`